RELEASE/scripts/autoscend/auto_bedtime.ash

```diff
@@ -755,14 +755,22 @@ boolean doBedtime()
 	{
 		item oreGoal = to_item(get_property("trapperOre"));
 		int need = 1;
+		boolean haveAdvSmithing = have_skill($skill[Super-Advanced Meatsmithing]);
 		if(oreGoal == $item[Chrome Ore])
 		{
 			need = 4;
 		}
-		if((item_amount($item[Chrome Ore]) >= need) && !possessEquipment($item[Chrome Sword]) && isArmoryAvailable())
+		if (!haveAdvSmithing) {
+			print('No Super-Advanced Meatsmithing for chrome sword crafting!');
+		}
+		if((item_amount($item[Chrome Ore]) >= need) && !possessEquipment($item[Chrome Sword]) && isArmoryAvailable() && haveAdvSmithing)
 		{
 			cli_execute("make " + $item[Chrome Sword]);
```

Comment:
this should really be a call to `create()` rather than cli_executing `make` so the return value can be checked for success but that's not your fault.